### PR TITLE
Create modal with Javascript

### PR DESCRIPTION
**Pull request by brahimdidi**
In this project, we will create a popup page using javascript and adding HTML and css by manipulating the dom.
## Requirements 
When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
When the user clicks (or taps) the close (X) button, the popup disappears.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -62,3 +62,72 @@ cancelbtn.addEventListener('click', cancel);
...
+const main = document.querySelector('.main');
+const footer = document.getElementById('footer');
+// cancelIcon Object
+const cancelIconInfos = {
+  src: 'template5Images/Icon-Cancel.svg',
+  alt: 'cancelIcon',
+};
+const Modalsection = document.createElement('section');
+Modalsection.classList.add('dis-none');
+Modalsection.innerHTML = `
```
Comment (oxenprogrammer) [inline main.js:77]: **Changes Required**
- [ ] **In order to associate each project with the popup window details, you must use a JavaScript array to store all of the information for all projects**. The html is currently hard-coded and you need to use JavaScript to create elements and dynamically append them to the DOM. Let us please try to fix this as specified in the project requirements.

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi **@brahimdidi**  and **@Ugboaja-Uchechi** , 

Good job so far!
I can see you're really putting effort to become a world-class software developer :+1:
There are some issues that you still need to address in order to move forward to the next project.
Trust me you got this! :muscle:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Consider the optional changes nice to have features and only work on them when you have satisfied the required changes_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: main.js [hunk 2]
```diff
@@ -62,3 +62,112 @@ cancelbtn.addEventListener('click', cancel);
...
+const footer = document.getElementById('footer');
+// cancelIcon Object
+const cancelIconInfos = {
+  src: 'template5Images/Icon-Cancel.svg',
+  alt: 'cancelIcon',
+};
+const card1 = cardInfos[0];
+// const card2 = cardInfos.card2;
+// const card3 = cardInfos.card3;
+// const card4 = cardInfos.card4;
```
Comment (rloterh) [inline main.js:116]: - [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi ,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢:shipit: . Congratulations!  🎉🌟 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comments marked **[OPTIONAL]** under this review:_
<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**